lib/themes/dosomething/paraneue_dosomething/templates/system/html.tpl.php

```diff
@@ -39,6 +39,26 @@
 </head>
 
 <body class="<?php print $classes; if ($variables['is_affiliate']) print ' -affiliate'; ?>" <?php print $attributes;?>>
+  <?php if ($variables['enable_facebook_login']): ?>
+    <script>
```

Comment:
This is intentionally loaded at the top per Facebooks documentation https://developers.facebook.com/quickstarts/105775762330/?platform=web

It loads async so it shouldnt block page rendering